genesis-tools/genesis-populate/src/lib.rs

```diff
@@ -65,6 +65,7 @@ impl GenesisBuilder {
             None,
             None,
             None,
+            None,
```

Comment:
That won’t compile. You need to pass the number.

It might be even better if we got rid of NightshadeRuntime::new and use NightshadeRuntime::with_config everywhere but that’s something for another PR.

Reply:
Let me change it to 5!